docs/tutorials/gs_continuous_space.ipynb

```diff
@@ -0,0 +1,1210 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.        distances = distances[...,idxs[0],idxs[1],:]
</pre><p>you already defined this above, so maybe you can reuse the definition from before?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/1574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>